src/main/java/com/tikal/jenkins/plugins/multijob/MultiJobBuild.java

```diff
@@ -201,16 +211,21 @@ public SubBuild(String parentJobName, int parentBuildNumber,
         public SubBuild(String parentJobName, int parentBuildNumber,
                 String jobName, int buildNumber, String phaseName,
                 Result result, String icon, String duration, String url,
-                boolean retry, boolean aborted) {
-            this.parentJobName = parentJobName;
+                boolean retry, boolean aborted, AbstractBuild jobBuild) {
+            this.parentJobName = Util.fixNull(parentJobName);

```

Comment:
I suppose null values should be considered as bug. If not, it makes sense to add <code>@CheckForNull</code> attributes to the parameter and the class field